src/plumpy/processes.py

```diff
@@ -697,6 +697,8 @@ def on_entering(self, state: process_states.State) -> None:
             call_with_super_check(self.on_except, state.get_exc_info())  # type: ignore
 
     def on_entered(self, from_state: Optional[process_states.State]) -> None:
+        from aio_pika.exceptions import ChannelInvalidStateError, ConnectionClosed
```

Comment:
Shouldn't these then not be removed from the top level? As far as I can see they are still being imported top level as well

Reply:
Weird, I removed them, but the merge commit put them back for some reason. I think I'll just close this PR since this needs a more hollistic approach.